cmake/toolchain/xtools/generic.cmake

```diff
@@ -0,0 +1,23 @@
+set_ifndef(XTOOLS_TOOLCHAIN_PATH "$ENV{XTOOLS_TOOLCHAIN_PATH}")
+set(       XTOOLS_TOOLCHAIN_PATH    ${XTOOLS_TOOLCHAIN_PATH} CACHE PATH "")
+assert(    XTOOLS_TOOLCHAIN_PATH     "XTOOLS_TOOLCHAIN_PATH is not set")
```

Comment:
spacing

Reply:
Fixed (vertically aligned the code for readability)